packages/gestalt/src/Callout.js

```diff
@@ -222,14 +250,15 @@ Callout.propTypes = {
   iconAccessibilityLabel: PropTypes.string.isRequired,
   message: PropTypes.string.isRequired,
   // $FlowFixMe[signature-verification-failure] flow 0.135.0 upgrade
-  primaryLink: PropTypes.shape({
-    href: PropTypes.string.isRequired,
+  primaryButton: PropTypes.shape({
+    href: PropTypes.string,
     label: PropTypes.string.isRequired,
     onClick: PropTypes.func,
     accessibilityLabel: PropTypes.string,
+    role: PropTypes.oneOf(['button', 'link']),
   }),
   // $FlowFixMe[signature-verification-failure] flow 0.135.0 upgrade
-  secondaryLink: PropTypes.shape({
+  secondaryButton: PropTypes.shape({
```

Comment:
Would `primaryAction` and `secondaryAction` make it more flexible and clear vs. specifying either Button or Link?

Reply:
I really like primary/secondaryAction! Updated!